.github/workflows/ci-stage.yml

```diff
@@ -0,0 +1,224 @@
+name: Build & deploy to staging
+
+on:
+  # run it during push to branch
+  push:
+    branches: [ infra_as_code_staging ]
```

Comment:
Will we be tracking pushes to this branch going forward to push code to staging.gitcoin.co

Reply:
We will! Appreciate you catching this!